pages/management/ManagementHandler.inc.php

```diff
@@ -181,19 +181,23 @@ function website($args, $request) {
 	}
 
 	/**
-	 * Display workflow settings
-	 * @param $args array
-	 * @param $request PKPRequest
-	 */
+	* Display workflow settings
```

Comment:
We actually do prefer to have the spacing the way it was. Purely aesthetic but for consistency's sake :)